.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: 🐛 Bug report
+description: Create a report to help us reproduce and fix a bug
+labels: [bug]
+
+body:
+- type: textarea
+  attributes:
+    label: Description
+    description: >
+      Please provide a clear and concise description of the bug.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Steps to reproduce
+    description: >
+      We can't solve your issue if we can't reproduce it. Please provide a
+      [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)
+      that shows how to reproduce the bug. If the bug requires any additional files to
+      reproduce, please upload those or provide a download link. Your code should be
+      runnable and include all relevant imports.
+    placeholder: |
+      1. Download any Landsat 8 scene from https://earthexplorer.usgs.gov/
+      2. Run the following code to reproduce the error
+
+      ```python
+      from torchgeo.datasets import Landsat8
+
+      ds = Landsat8("/path/to/downloads/directory")
+      ...
+      ```
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Version
```

Comment:
Instead of just the TorchGeo version, we could expand this to the full development environment, including:

* OS: e.g., macOS 12.4
* Python: e.g., 3.9.12
* TorchGeo: e.g., 1.2.3.dev0 (e1285e6cc6b65080e82bdbf7de9dea3f647d8b3b)
* Installation method: e.g., pip, conda, spack, git, etc.
* Dependency versions: e.g., `pip list`, `conda list`, `spack find`, etc.

I'm not sure how useful the rest of these are since TorchGeo is pure-Python and pretty heavily tested...

Reply:
Yeah I think torchgeo version is fine for now (i.e. until we see something crazy that is not caught by tests or explained by dependency/Python version)